lib/analytics/course_articles_csv_builder.rb

```diff
@@ -87,6 +90,12 @@ def character_sum(article_data)
     end
   end
 
+  def references_count(article_data)
+    article_data[:references] do |sum, references|
+      references ? sum + references : sum
```

Comment:
I don't understand what's going on here. What does `article_data[:references]` look like, and where does the `sum` come in?

Reply:
Sorry, was just checking it, forgot to remove it